Translations update from Translate H5P

translate-h5p opened the pull request:
Translations update from [Translate H5P](https://translate-h5p.tk) for [H5P/h5p-content-calendar](https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/).



Current translation status:

![Weblate translation status](https://translate-h5p.tk/weblate/widget/h5p/h5p-content-calendar/horizontal-auto.svg)

otacke reviewed (changes requested):
Rejected due to empty objects in files an uppercase letters in file name.

weblate committed:
Prefill add-on <noreply-addon-prefill@weblate.org> updated Spanish translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/es/

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translation: H5P/h5p-content-calendar
Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/

Antonio Aneiros <antonio.aneiros@gmail.com> deleted Spanish translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/es/

weblate committed:
Prefill add-on <noreply-addon-prefill@weblate.org> updated Spanish (Mexico) (es-MX) (es-MX) translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/es-MX/

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translation: H5P/h5p-content-calendar
Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/

German Valero <gvalero@unam.mx> deleted Spanish (Mexico) (es-MX) (es-MX) translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/es-MX/

weblate committed:
Prefill add-on <noreply-addon-prefill@weblate.org> updated Galician translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/gl/

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translation: H5P/h5p-content-calendar
Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/

Antonio Aneiros <antonio.aneiros@gmail.com> deleted Galician translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/gl/

weblate committed:
Prefill add-on <noreply-addon-prefill@weblate.org> updated Lithuanian translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/lt/

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translation: H5P/h5p-content-calendar
Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/

Gintarė <gintare.lose@ktu.lt> deleted Lithuanian translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/lt/

weblate committed:
Prefill add-on <noreply-addon-prefill@weblate.org> updated Basque translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/eu/

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translation: H5P/h5p-content-calendar
Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/

Iñigo Zendegi Urzelai <izendegi@mondragon.edu> deleted Basque translation using Weblate @ translate-h5p.tk.

 Translate-URL: https://translate-h5p.tk/weblate/projects/h5p/h5p-content-calendar/eu/

otacke committed:
Rename es-MX.json to es-mx.json
